example/example.js

```diff
@@ -17,6 +17,16 @@ const config = {
   }
 };
 
-app.use('/', SensorThings(config));
+const clientHooks = {
+  'Things': {
+    'beforeCreate': (instance, options) => {
```

Comment:
You should be able to get the entity model by inspecting `instance.$modelOptions.name.plural`. Like in https://github.com/mozilla-sensorweb/sensorthings/blob/4a101338acd789dfd78f143c7c81560392db4544/src/models/association.js#L227 for example.

```javascript
const hooks = {
  'beforeCreate': instance => {
     console.log(instance.$modelOptions.name);
  }
};
```